packages/modelviewer.dev/src/docs-and-examples/sidebar.ts

```diff
@@ -112,6 +112,7 @@ function updateSidebarView(
 
 /*
  * Hide all of the entries not within the current subcategory
+ * entries should be every entry on the page when this is called
```

Comment:
Try to make comments complete sentences with punctuation and such.

Reply:
will do in next pr!